source/Octopus.Client.Tests/AutomationEnvironment/AutomationEnvironmentProviderFixture.cs

```diff
@@ -0,0 +1,51 @@
+using NUnit.Framework;
+using Octopus.Client.AutomationEnvironments;
+using Octopus.Client.Model;
+
+namespace Octo.Tests.Commands
+{
+    class AutomationEnvironmentProviderFixture
```

Comment:
This is a slight re-write of `ServiceMessagesTestFixture` from Octopus.Cli, as it needed to refer to internal props.